Formula/libhandy.rb

```diff
@@ -89,13 +89,18 @@ def install
       -lgio-2.0
       -lglib-2.0
       -lgobject-2.0
-      -lgtk-3.0
+      -lgtk-3
```

Comment:
Why not
```suggestion
      -lgtk-#{gtk.version.major}
```

Reply:
Reason: yes

Reply:
It seems we hardcode version for most GNOME formulae right now. The other parts of libs/includes are the same. Not sure how much changes when going from GTK versions like `gtk+ --> gtk+3 --> gtk4`.

For now, I think we can leave it as the output seems to be just copied from `pkg-config gtk+-3.0 --libs`.

In future, we could consider refactoring to add a `pkg-config` or `cmake` test dependency to just let tools pick up dependencies using `*.pc` or `*.cmake` files.

Reply:
I'm pretty sure that because of the way GNOME interacts with GTK, we have to build all the formulae to the same version of GTK.  Once we have all the GNOME formulae bottled using GTK 3 on Linux (I think they are already all bottled for macOS), we could then move them all to GTK 4 if they officially support it.